sdk/identity/Azure.Identity/src/Credentials/AzureDeveloperCliCredential.cs

```diff
@@ -123,7 +126,7 @@ private async ValueTask<AccessToken> RequestCliAccessTokenAsync(bool async, Toke
                 ScopeUtilities.ValidateScope(scope);
             }
 
-            GetFileNameAndArguments(context.Scopes, tenantId, out string fileName, out string argument);
+            GetFileNameAndArguments(context.Scopes, tenantId, context.Claims, out string fileName, out string argument);
```

Comment:
`azd` [expects a base64 encoded value](https://github.com/Azure/azure-dev/blob/a9a799fcecb24305bdf57298453574e3b0ac9622/cli/azd/cmd/auth_token.go#L168-L173) for `--claims` while [documentation for TokenRequestContext.Claims](https://learn.microsoft.com/en-us/dotnet/api/azure.core.tokenrequestcontext.claims?view=azure-dotnet#azure-core-tokenrequestcontext-claims) implies its value should be a JSON string. Does the auth policy decode claims before calling `GetToken`?

Reply:
👍 then we need to encode them here